dvc/remote/gs.py

```diff
@@ -87,33 +86,21 @@ def remove(self, path_info):
 
         blob.delete()
 
-    def _list_paths(self, bucket, prefix, gs=None):
-        gs = gs or self.gs
+    def _list_paths(self, bucket, prefix, ctx=None):
+        gs = ctx or self.gs
 
         for blob in gs.bucket(bucket).list_blobs(prefix=prefix):
             yield blob.name
 
     def list_cache_paths(self):
         return self._list_paths(self.path_info.bucket, self.path_info.path)
 
-    def exists(self, path_info):
-        paths = set(self._list_paths(path_info.bucket, path_info.path))
+    def exists(self, path_info, **kwargs):
+        paths = set(
+            self._list_paths(path_info.bucket, path_info.path, **kwargs)
```

Comment:
should it be cxt there?

Reply:
It is there, we are just passing it through to `_list_paths`.

Reply:
yeah, but do we need kwargs or one cxt var is enough?

Reply:
cxt is enough, but kwargs is more generic.

Reply:
I'm not sure it's worth generalizing ahead of time, until there it's clear that it's necessary

Reply:
Makes sense, will adjust.